### DISCUSSION TITLE
Intersection between edges

### DISCUSSION THREAD

**Discussion by zairiint** [imixs/open-bpmn#211] [Q&A]
Hello,

I'm working in a complex diagram view with multiple nodes so that there is a lots of intersections between edge.

Did someone have a solution/example to avoid those intersections like picture bellow 

thanks in advance!

![capture1](https://user-images.githubusercontent.com/98454096/224078364-85219a01-03b7-4612-bb68-02f7ad61eef9.PNG)

**Comment by rsoika** [comment]
Hi @zairiint 
this topic is handled by the [sprotty framework](https://github.com/eclipse-sprotty/sprotty) There is a very powerful routing concept which can be adapted. The Router is responsible to compute the intersection. And the Edge-View is responsible for the display.

You should ask your question directly by opening a issue here: https://github.com/eclipse-sprotty/sprotty/issues

Eclipse-GLSP and Open-BPMN is adapting this technology. 

https://github.com/eclipse-sprotty/sprotty
